eb

```diff
@@ -39,48 +39,35 @@
 # @author: Pieter De Baets (Ghent University)
 # @author: Jens Timmerman (Ghent University)
 
+PYTHON=python
+
 # Python 2.6 or more recent 2.x required
 REQ_MAJ_PYVER=2
 REQ_MIN_PYVER=6
 REQ_PYVER=${REQ_MAJ_PYVER}.${REQ_MIN_PYVER}
 
 # make sure Python version being used is compatible
-pyver=`python -V 2>&1 | cut -f2 -d' '`
+pyver=`$PYTHON -V 2>&1 | cut -f2 -d' '`
 pyver_maj=`echo $pyver | cut -f1 -d'.'`
-pyver_min=`echo $pyver | cut -f2 -d'.'`
+
+if [ $pyver_maj -ne $REQ_MAJ_PYVER ]
```

Comment:
Well, except that it's different from what we're doing now, and may lead to surprises...

What if `python` (Py2.x) is available in one place, and python2 in another location (and they're not from the same Python installation)?

Right now, we would use `python`, but with your proposed change we would use `python2`...